source/Account.h

```diff
@@ -54,6 +54,7 @@ class Account {
 	const std::vector<Mortgage> &Mortgages() const;
 	void AddMortgage(int64_t principal);
 	void AddFine(int64_t amount);
+	void AddDebt(int64_t amount,int term=-1,double interest=-1,const std::string &type="");
```

Comment:
Spacing around commas and operators. Also, will this function ever be called without a type specified? If not, then that parameter should not be defaulted.